Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1633,4 +1633,4 @@ in great details:
       <https://docs.google.com/presentation/d/1VOUcoIxgM_c6M_zAV1dLlRCjyYCMdR3tJv6CEdfLMh8/edit>`__,
       `video <https://fosdem.org/2020/schedule/event/containers_bpf/>`__)
     * "Kernel improvements for Cilium socket LB" (LSF/MM/BPF 2020, `slides
-      <https://docs.google.com/presentation/d/1w2zlpGWV7JUhHYd37El_AUZzyUNSvDfktrF5MJ5G8Bs/edit#slide=id.g746fc02b5b_2_0>`__)
```

Comment:
It's nice to have the anchors to the right slide, though. I think we can ignore that by configuring the right option for the link checker? I found an old branch in my local repo, where I had this:

```python
linkcheck_anchors_ignore = [
    [...]
    'slide=id.g',
    [...]
]
```

(Actually, aren't these supposed to be skipped given that you have `linkcheck_anchors = False`?)

Same comment for other links to Google presentations.

Reply:
Yes, they will be ignored by the `linkcheck_anchors`, but when I went to the url the slide id was the id of the first slide. So I removed it from the url.
I can add those back and ignore the ids in `linkcheck_ignore`.

I initially tried to make all the anchors work (the ones that were possible) and added `linkcheck_achor=False` after.

Reply:
OK, please double-check in this case. Maybe some of the anchors point to the first slides, but when I tried the other day, I'm pretty sure at least one of them was pointing to as slide in the middle of the deck. Using `linkcheck_anchors_ignore` seems like a good solution to keep the anchors working; worst case we can ignore them, but I'd rather avoid removing the anchors if we need to point to a particular slide.